src/pykeen/evaluation/ogb_evaluator.py

```diff
@@ -61,6 +83,15 @@ def evaluate_ogb(
     if batch_size is None:
         raise NotImplementedError("Automatic batch size selection not available for OGB evaluation.")
 
+    additional_filter_triples = kwargs.pop("additional_filter_triples", None)
+    if additional_filter_triples is not None:
+        # fixme: this is not an elegant solution; it would be better to encapsulate filter triples in the evaluator instance
+        logger.warning(
+            f"evaluate_ogb received additional_filter_triples={additional_filter_triples}. However, it uses "
+            f"explicitly given filtered negative triples. It ignores these instead of throwing an error to ensure "
+            f"compliance with the current way the pipeline calls evaluators. This may change in the future."
+        )
```

Comment:
@cthoyt what's your opinion on this? Should be block this PR until we find a satisfying solution here? Or would you rather prefer having this evaluator somewhat useable sooner (you would still need to explicitly provide precomputed negative samples for the constructor)?

Reply:
nope I think that having some logging is good enough - maybe we can also have a note that this is experimental and needs some more work in the docstring.